src/index.js

```diff
@@ -1,22 +1,7 @@
+// eslint-disable-next-line no-unused-vars
 import _ from 'lodash';
 import './style.css';
-import Icon from './icon.png';
+// import pokemon from './img/pokemon.jpg';
```

Comment:
Please do not use unused code as a comment, because it shows your code is not clean.